templates/epub3/toc.xhtml.ejs

```diff
@@ -11,9 +11,21 @@
 <nav id="toc" epub:type="toc">
     <ol>
         <% content.forEach(function(content, index){ %>
+            <% if(!content.excludeFromToc && content.beforeToc){ %>
+                <li class="table-of-content">
+                    <a href="<%= content.href %>"><%= (content.title || "Chapter "+ (1+index)) %><% if(content.author.length){ %> - <small class="toc-author"><%= content.author.join(",") %></small><% }%><% if(content.url){ %><span class="toc-link"><%= content.url %></span><% }%></a>
+                </li>
+            <% } %>
+        <% }) %>
         <li class="table-of-content">
-            <a href="<%= content.href %>"><%= (content.title || "Chapter "+ (1+index)) %><% if(content.author.length){ %> - <small class="toc-author"><%= content.author.join(",") %></small><% }%><% if(content.url){ %><span class="toc-link"><%= content.url %></span><% }%></a>
+            <a href="toc.xhtml">- <%= tocTitle %> -</a>
```

Comment:
Hi @cyrilis, yap I think that proposal of yours seems a bit better and clear to me than what we have currently. I would just suggest for the type to be lowercase, like so `type: 'toc'`. 🙂